system/Router/Router.php

```diff
@@ -548,11 +548,13 @@ public function autoRoute(string $uri)
 			$this->params = $segments;
 		}
 
+		$defaultNameSpace = $this->collection->getDefaultNamespace();
```

Comment:
I know this is a bit picky, but can we make it `defaultNamespace`?

Reply:
I, also, think this is picky and would like to see it changed 😅

Reply:
I've updated to `defaultNamespace` 👍